GameServerLib/Items/Inventory.cs

```diff
@@ -130,7 +143,13 @@ private IItem AddStackingItem(IItemData item)
             }
             return AddNewItem(item);
         }
+        public void RemoveStackingItem(Game game, string itemSpellName, IObjAiBase owner)
+        {
+            IItem item = GetItem(itemSpellName);
 
+            game.PacketNotifier.NotifyRemoveItem(owner, GetItemSlot(item), (byte)(item.StackCount - 1));
```

Comment:
Why not Decrement before the packet?

Reply:
Because it would cause a crash with 1 stack, since the item would be removed, resulting on a null exception

Reply:
Resolved then.

Reply:
So, i found out that it wasn't crashing because the item was null, is was because it was casting a BaseSpell